meteor-app/imports/ui/layouts/ordering.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Sidebar, Menu, Icon } from 'semantic-ui-react'
+import { Switch, Route } from 'react-router-dom'
+import NotFound from '/imports/ui/not-found'
+import OrderingMainContainer from '/imports/ui/ordering/ordering-main-container'
+import '/imports/ui/layouts/attendance.css'
+import Nav from '/imports/ui/member/member-nav'
+import Alert from 'react-s-alert';
```

Comment:
I know you didn't introduce this, but it concerns me that we need to import 3 files for one component. And this will be repeated every time we use an alert.

If you read the documentation here https://github.com/juliancwirko/react-s-alert
you will see that the config and one instance needs to be imported in the main app.

Any calls to alert should be done programmatically. so we'll only need 1 import, and one line of code to create the alert